### ISSUE TITLE
Linux SystemPrep Output-logging breaks script-execution

### ISSUE THREAD

**Issue by ferricoxide** [lorengordon/SystemPrep#25]
Looks like you updated the `SystemPrep-Bootstrap--Linux.sh` a few days ago, but when it's called via userdata, I find:

~~~
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
^M  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0^M100 11386  100 11386    0     0  11290      0  0:00:01  0:00:01 --:--:-- 62906
sh: line 39: syntax error near unexpected token `>'
2015-08-12 18:28:23,007 - util.py[WARNING]: Failed running /var/lib/cloud/instance/scripts/part-001 [2]
2015-08-12 18:28:23,010 - cc_scripts_user.py[WARNING]: Failed to run module scripts-user (scripts in /var/lib/cloud/instance/scripts)
~~~

in the cloud-init logs (`/var/tmp/cloud-init-output.log`). I attempted to manually rerun the copy that cloud-init stashes in `/var/lib/cloud/instance/scripts` but got a similar error. Then tried to manually re-run it:

~~~
# /usr/bin/curl -s -L https://s3.amazonaws.com/systemprep/BootStrapScripts/SystemPrep-Bootstrap--Linux.sh | sh
sh: line 39: syntax error near unexpected token `>'
~~~

Wanted to verify it wasn't simply an incompatibility with the invocation-method, so I tried a download-and-execute attempt:

~~~
# /usr/bin/curl -s -L https://s3.amazonaws.com/systemprep/BootStrapScripts/SystemPrep-Bootstrap--Linux.sh > /tmp/blah && sh /tmp/blah
/tmp/blah: line 39: syntax error near unexpected token `>'
~~~

Looking at the called-out line:

~~~
# sed -n '39p' /tmp/blah
exec > >(tee "${LOGFILE}" | "${LOGGER}" -i -t "${LOGTAG}" -s 2> /dev/console) 2>&1
~~~

It would appear that the setup of the multi-outputter is making it choke. Given the update date (and associated commit-message), is it possible that the source-script is fixed but the bucket wasn't properly updated?

**Comment by ferricoxide** [comment]
Basically: was wanting to stand up an instance with SystemPrep already run so that I could add the requisite file_roots directive to begin supporting Salt'ing HBSS install.

**Comment by lorengordon** [comment]
That line hasn't changed in *months*. The timestamps seem to indicate that I updated the bucket right after making the last change to the file in Github 5 days ago. What's in the bucket and what's in the repo should be exactly the same. And I've provisioned several instances since from that url over the last 5 days. So, not sure what might be happening... I'll give it a spin when I get home though.
